### PR TITLE
Update to VS 2017

### PR DESCRIPTION
**Pull request by cijothomas**

### CODE REVIEW

File: test/MVCFramework45.FunctionalTests/Views/_ViewImports.cshtml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-﻿@using MVCFramework45.FunctionalTests
+﻿Moi@using MVCFramework45.FunctionalTests
```
Comment (pharring) [inline test/MVCFramework45.FunctionalTests/Views/_ViewImports.cshtml]: Keyboard slip?

Comment (cijothomas) [inline test/MVCFramework45.FunctionalTests/Views/_ViewImports.cshtml]: Good find - corrected.

File: src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj [hunk 2]
```diff
@@ -0,0 +1,84 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Application Insights for ASP.NET Core web applications. See https://azure.microsoft.com/en-us/documentation/articles/app-insights-asp-net-five/ for more information. Privacy statement: https://go.microsoft.com/fwlink/?LinkId=512156</Description>
```
Comment (SergeyKanzhelev) [inline src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj]: remove `en-us` from the link

File: src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    <Copyright>Copyright © Microsoft. All Rights Reserved.</Copyright>
+    <AssemblyTitle>Application Insights for ASP.NET Core Web Applications</AssemblyTitle>
+    <VersionPrefix>2.1.1</VersionPrefix>
+    <Authors>Microsoft</Authors>
+    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
```
Comment (SergeyKanzhelev) [inline src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj]: are you sure about 451? I thought aspnetcore is 46+

Comment (SergeyKanzhelev) [inline src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj]: 46 is better for correlation

File: test/MVCFramework45.FunctionalTests/MVCFramework45.FunctionalTests.csproj [hunk 4]
```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <TargetFrameworks>net451;netcoreapp1.0</TargetFrameworks>
```
Comment (SergeyKanzhelev) [inline test/MVCFramework45.FunctionalTests/MVCFramework45.FunctionalTests.csproj]: name of the project says 45. Do you need netcore target then?

File: src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj (hunk 3 above)
Comment (cijothomas) [inline src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj]: Not changing anything in this PR, as this is simply converting from project.json to .csproj model.

File: test/MVCFramework45.FunctionalTests/MVCFramework45.FunctionalTests.csproj (hunk 4 above)
Comment (cijothomas) [inline test/MVCFramework45.FunctionalTests/MVCFramework45.FunctionalTests.csproj]: Sounds incorrect maybe. But wont be changing in this PR as  this is simply migrating project.json to .csproj.

**Pull request merged**